### DISCUSSION TITLE
Remote live reload or hot reload over Websocket

### DISCUSSION THREAD

**Discussion by namanh6669** [webpack/webpack#16592] [Q&A]
I was deployed a project to host like "example.com" or "10.x.x.121:9000"
I want send my changes (code, files,...) from "localhost:9000" to that host above and live reload or hot reload over WebSocket.

Please tell me how to do that? Or Do have any ideal like write custom plugin or custom websocketserver,websocketclient or some any way to do that?.

Thanks!

**Comment by alexander-akait** [comment]
Why you deploy development application? You can use nginx or another proxt server to handle such case

**Reply by namanh6669** [reply]
Because i need development env to tester can test on that. But i am finding solution to quick update it from local without rebuild my application.

**Reply by namanh6669** [reply]
@alexander-akait Can you support me this case ?

**Reply by alexander-akait** [reply]
In such case you can use production public path and setup a proxy server (nginx or using proxy option for dev server and rewrite request)
